docs/fastdds/property_policies/non_consolidated_qos.rst

```diff
@@ -13,22 +13,24 @@ to a different release version.
 
 .. _push_mode_qos_policy:
 
-``Push Mode`` QoS Policy
-^^^^^^^^^^^^^^^^^^^^^^^^
+``DataWriter operating mode`` QoS Policy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-By default, Fast DDS DataWriters are enabled using the ``push mode``.
-This implies they will send new samples to matched readers immediately after adding them to their queue.
+By default, *Fast DDS* |DataWriters| are enabled using ``push mode``.
+This implies that they will add new samples into their queue, and then immediately deliver them to matched readers.
 For writers that produce non periodic bursts of data, this may imply saturating the network with a lot of packets,
 increasing the possibility of losing them on unreliable (i.e. UDP) transports.
 Depending on their QoS, DataReaders may also have to ignore some received samples, so they will have to be resent.
 
 Configuring the DataWriters on ``pull mode`` offers an alternative by letting each reader pace its own data stream.
-It works by notifying the reader what it is missing, and wait for it to request only as much as it can handle.
-At the cost of greater latency, this model can deliver reliability while using far fewer packets than in ``push mode``.
+It works by the writer notifying the reader what it is available, and wait for it to request only as much as it can
```

Comment:
```suggestion
It works by the writer notifying the reader what it is available, and waiting for it to request only as much as it can
```

Reply:
Done in 2535f2ab210ae79568ed2291a522519233bca5df